jax_tqdm/pbar.py

```diff
@@ -1,13 +1,16 @@
 import typing
 
 import jax
+import tqdm.auto
+import tqdm.notebook
+import tqdm.std
 from jax.debug import callback
-from tqdm.auto import tqdm
 
 
 def scan_tqdm(
     n: int,
     print_rate: typing.Optional[int] = None,
+    tqdm_type: str = "auto",
```

Comment:
Could be useful to have a check here that `tqdm_type` is one of the options, may make the error clearer if someone passes an incorrect string when it tries to get the attribute later

Reply:
Added a simple check